spec/codecs/multiline_spec.rb

```diff
@@ -157,4 +157,52 @@
 
     end
   end
+
+  context "with non closed multiline events" do
+    let(:random_number_of_events) { rand(300..1000) }
+    let(:sample_event) { "- Sample event" }
+    let(:events) { decode_events }
+
```

Comment:
It will be random on each test run / jenkins, if you do multiple loops this will make test harder to reproduce locally even if you have the seed. This is what I believe.